packages/itwin/map-layers/src/public/locales/en/mapLayers.json

```diff
@@ -169,10 +169,15 @@
     "SetTransparency": "Set transparency"
   },
   "WellKnownBaseMaps": {
-    "BingProvider": {
+    "BingMaps": {
```

Comment:
i was expecting more changes from renaming this key

Reply:
Those keys are used to map a setting object to its localization strings.  We now rely on the format id instead of the provider name.